### PR TITLE
[client] Add more fields to Client Entities

### PR DESCRIPTION
**Pull request by captn3m0**
MICR/SWIFT/NEFT/UPI/IMPS/RTGS/CENTRE fields were missing in some clients.

- [x] Ruby
- [x] PHP
- [x] Golang
- [x] Javascript

TODO:

- [x] Add tests
- [x] Update docs

### CODE REVIEW

File: src/go/client_test.go [hunk 1]
```diff
@@ -111,7 +110,7 @@ func TestLookUP(t *testing.T) {
 				assert.Equal(t, tt.IfscResponse.NEFT, got.NEFT)
 				assert.Equal(t, tt.IfscResponse.SWIFT, got.SWIFT)
 				assert.Equal(t, tt.IfscResponse.IMPS, got.IMPS)
-				assert.Equal(t, tt.IfscResponse.BankCode, got.BankCode)
+				// assert.Equal(t, tt.IfscResponse.BankCode, got.BankCode)
```
Comment (gyanesh-m) [inline src/go/client_test.go:113]: why ?

Comment (captn3m0) [reply src/go/client_test.go]: Had commented while testing something, fixed.

File: tests/node/client_test.js [hunk 2]
```diff
@@ -14,6 +16,16 @@ ifsc
     assert.equal('KKBK0000261',res['IFSC'])
     assert.equal('110485003',res['MICR'])
     assert.equal('HARYANA',res['STATE'])
+    assert.equal('110485003',res['MICR'])
```
Comment (gyanesh-m) [inline tests/node/client_test.js:19]: add assertion for upi,rtgs, neft and imps.

Comment (captn3m0) [reply tests/node/client_test.js:19]: Were covered in the second test, but added anyway.

**Pull request merged**